.gitignore

```diff
@@ -0,0 +1,2 @@
+# Ignore test.md
+test.md
```

Comment:
- [ ] It is better to also add `node_modules/` as one of the files/directory to ignore as it is a requirement. Node modules directory may stall git operations and make your Github look messy. It could be accidentally added to your project by various installations.

Reply:
Thank you for your feedback. I will work on it as soon as possible